vortex-error/src/lib.rs

```diff
@@ -68,71 +68,74 @@ impl From<Infallible> for VortexError {
     }
 }
 
+const _: () = {
+    assert!(size_of::<VortexError>() < 128);
+};
```

Comment:
One lint that popped off when I switched to stable was that apparently with all features activated, VortexError was a whopping 140 bytes, which was tripping the large_err check.

I boxed the Backtraces to keep the size down, now it's around ~88B. Still large but better.

Reply:
Yes but this is not new, the lint is just new.

Reply:
Filed #3892 as a follow up